schemas/2017-12-01/Microsoft.DBforPostgreSQL.json

```diff
@@ -532,11 +473,12 @@
                 "9.6",
                 "10",
                 "10.0",
-                "10.2"
+                "10.2",
```

Comment:
Same here. We don't want to expose 10.2

Reply:
(copying previous comment)
Swagger is the main API spec for your RP - technically you're already declaring support for 10.2 here. If you need to fix this then I'd suggest submitting a PR to this repo:
https://github.com/Azure/azure-rest-api-specs/tree/master/specification/postgresql/resource-manager/Microsoft.DBforPostgreSQL